packages/ckeditor5-dev-utils/lib/bundler/getlicensebanner.js

```diff
@@ -10,10 +10,10 @@ module.exports = function getLicenseBanner() {
 
 	/* eslint-disable indent */
 	return (
-`/**
- * @license Copyright (c) 2003-${ date.getFullYear() }, CKSource - Frederico Knabben. All rights reserved.
- * For licensing, see LICENSE.md.
- */`
+`/*--------------------------------------------------------------------------------------*
```

Comment:
It requires a comment on why it needs to look this way.

Reply:
We can think about a shorter comment or some short custom plugin for the banner.

Reply:
BTW can't we look for `/\/\*\n \* @license/`? And use the old license comment?

Reply:
Because once you wrote about a shorter comment I thought that it unnecessarily increases a size of every build. Not by much, perhaps it gzips well, but it seems unnecessary if the regexp can look for `@license` too.

Reply:
Doh, we use `@license` in every single source file :D

Reply:
See https://github.com/ckeditor/ckeditor5-dev/pull/373#discussion_r199095445 too 😄

Reply:
After a couple of deep breaths, I'm for extending `CKEditorWebpackPlugin` so it adds this banner in a way which doesn't conflict with minification.

This way, we'll simplify maintaining configurations. We'll be able to get rid of this:

```js
		new webpack.BannerPlugin( {
			banner: bundler.getLicenseBanner(),
			raw: true
		} ),
```

And will be able to avoid adding this:

```js
new UglifyJsWebpackPlugin( {
		sourceMap: true,
		uglifyOptions: {
			output: {
				comments: /^\**!/
			}
		}
	} )
```

So, `CKEditorWebpackPlugin()` should have the following option:

`[options.addLicense=false] {Boolean|String}`

* `false` – don't add
* `true` – add the default one
* `String` – if any of sub-CKEditor projects will require a bit different license text

Additionally, we may need to rethink configuration because it will get crowded:

```js
		new CKEditorWebpackPlugin( {
			// Main language that will be built into the main bundle.
			language: 'en',

			// Additional languages that will be emitted to the `outputDirectory`.
			// This option can be set to an array of language codes or `'all'` to build all found languages.
			// The bundle is optimized for one language when this option is omitted.
			additionalLanguages: 'all',

			// Optional directory for emitted translations. Relative to the webpack's output.
			// Defaults to `'translations'`.
			// outputDirectory: 'ckeditor5-translations',

			// Whether the build process should fail if an error occurs.
			// Defaults to `false`.
			// strict: true,

			// Whether to log all warnings to the console.
			// Defaults to `false`.
			// verbose: true
		} ),
```

Reply:
I'll move this to the ticket cause it'd be pity to lose this.